src/components/CountrySVG/countries/CountryAD.vue

```diff
@@ -2,12 +2,24 @@
 import { useCountryGuessed } from '@/composables/useCountryGuessed.ts'
 import { computed } from 'vue'
 
-const { guessed } = useCountryGuessed('AD')
-const classes = computed(() => ({ 'fill-stone-400 stroke-black': guessed.value }))
+const { guessed, isLatestCountryGuessed } = useCountryGuessed('AD')
+
+const classes = computed(() => {
+  if (isLatestCountryGuessed.value) {
+    return 'fill-red-400 stroke-black'
+  }
+
+  if (guessed.value) {
+    return 'fill-stone-400 stroke-black'
+  }
```

Comment:
We should have in mind that if this stay the same, we can refactor this to abstract these classes into a composable as well. Maybe even abstract it into `useCountryGuessed` composable